vendor/golang.org/x/sys/unix/README.md

```diff
@@ -89,7 +89,7 @@ constants.
 
 Adding new syscall numbers is mostly done by running the build on a sufficiently
 new installation of the target OS (or updating the source checkouts for the
-new build system). However, depending on the OS, you may need to update the
+new build system). However, depending on the OS, you make need to update the
```

Comment:
Wouldn't "may" be correct here?

Reply:
Hi @seqdan  The files in vendor  folder  are dependencies, we cannot change this files.  We update this files with go command `go mod vendor` and `go mod tidy`.

Reply:
Oh, I understand! Sorry! - I'm observing this exciting project, but my skills are not sufficient to contribute. Apologies.

Reply:
Hi @seqdan , No need apologies. It's an opensource project, any questions and contributions is welcome. You can find the [contribute document](https://github.com/terraform-providers/terraform-provider-azuredevops/blob/master/docs/contributing.md) here.